### DISCUSSION TITLE
fre:ac auto-rip and auto-eject options

### DISCUSSION THREAD

**Discussion by Johnmhopkins57** [enzo1982/freac#595] [Help & Support]
In the previous century, fre:ac was the standard for my Windows days;
Have since "migrated" to Zorin OS 17 (Ubuntu 22.04/Linux);
Previous .flac collection was corrupted (roughly 200GB)
I have downloaded/installed fre:ac from Zorin "store" (Flathub).
During said Windows days, fre:ac had a feature for "automated ripping"... as well as automatic eject when done.
How to enable these options?
With Roughly 350 Audio CDs to rip (and some other stuff: DSD, DVD/blueray soundtracks)
Would really, really, really appreciate the solution.
Thank you,
john

**Comment by merryjhon94** [comment]
Automated features like auto-rip and auto-eject options enhance user convenience in devices, streamlining operations with minimal manual intervention. These functions not only save time but also ensure smoother operation. For businesses seeking efficiency without breaking the bank, consider employing [cheap ghostwriters for hire](https://www.americanauthorhouse.com/ghostwriting-services/) to craft compelling content.

**Reply by Johnmhopkins57** [reply]
![Ripper-Settings-Automatization](https://github.com/enzo1982/freac/assets/160749949/c8afcf35-2ada-4149-afa8-6deac10608ff)
merryjhon94 is a **USELESS SPAM** response.

As you can see in the screen grab: Ripper>Settings> Automatization,
"Read CD Contents On Insert" **and** "Start Ripping Automatically" is not a select-able option...
"Eject Disk After Ripping" functions as it should
fre:ac V1.1.7 was downloaded/installed  from the Zorin "store" ("flatpack")
BTW, have also set fre:ac as the default application for "music"
(for some odd reason, Zorin persists in "proposing" Rhythmbox, when inserting Audio CDs)

**Reply by calinadamut** [reply]
First, make sure you have an audio CD inserted in the drive
Otherwise, the Ripper/Settings submenu will not appear.

**Comment by dkowis** [comment]
Auto rip does not seem to work in the flatpak. It's probably a flatpak permissions thing. Using the AppImage downloaded from github, I was able to have it autoRip my CDs for me :D
